lib/Utils/LLVMShorthands.cpp

```diff
@@ -246,13 +246,13 @@ std::string getMetaDataID(const llvm::Value *V) {
   return "-1";
 }
 
-llvmValueIDLess::llvmValueIDLess() : sless(stringIDLess()) {}
+LLVMValueIDLess::LLVMValueIDLess() : Sless(StringIDLess()) {}
```

Comment:
Should this simple ctor be in the header? Would allow trivial inlining when constructing